api/src/signals/integrations/apptimize/handler.py

```diff
@@ -0,0 +1,61 @@
+import json
+
+from django.conf import settings
+from django.core.mail import send_mail
+
+from signals.models import Signal
+
+
+def is_signal_applicable(signal: Signal):
+    """Is given `Signal` applicable for Apptimize.
+
+    Note, this logic isn't tenable anymore.. The concept `categories` needs
+    to be refactored soon. Take a look at `signals.messaging.categories` as
+    well.
+
+    :param signal: Signal object
+    :returns: bool
+    """
+    eligible_main_categories = ('Openbaar groen en water',
+                                'Wegen, verkeer, straatmeubilair',
+                                'Afval',)
+    all_sub_categories_of_main_categories = (
+            settings.SUB_CATEGORIES_DICT['Openbaar groen en water'] +
+            settings.SUB_CATEGORIES_DICT['Wegen, verkeer, straatmeubilair'])
+
+    eligible_sub_categories = ()
+    for sub_category in all_sub_categories_of_main_categories:
+        eligible_sub_categories += (sub_category[2],)
+    eligible_sub_categories += ('Prullenbak is vol', 'Veeg- / zwerfvuil',)
+
+    is_applicable_for_apptimize = (
+            settings.EMAIL_APPTIMIZE_INTEGRATION_ADDRESS is not None
+            and signal.category.main in eligible_main_categories
+            and signal.category.sub in eligible_sub_categories)
+
+    return is_applicable_for_apptimize
+
+
+def handle(signal: Signal) -> None:
+    print('SENDING')
```

Comment:
`print` statement can be removed. If you really need it lets use info logging.

Reply:
done